### ISSUE TITLE
error while creating asset

### ISSUE THREAD

**Issue by SherinKR** [resilient-tech/india-compliance#1398]
### Information about bug

It's getting the following error while creating an Asset
![image](https://github.com/resilient-tech/india-compliance/assets/43608142/bee70f69-fbd5-4955-986d-4c151b20ca9b)

### Version

Frappe Version : 14.59.0
ERPNext Version : 14.54.0
India Compliance Version : 14.19.6

### Relevant log output / Stack trace / Full Error Message.

```shell
File "apps/erpnext/erpnext/assets/doctype/asset/asset.py", line 45, in validate
18:09:16 web.1            |     self.prepare_depreciation_data()
18:09:16 web.1            |   File "apps/erpnext/erpnext/assets/doctype/asset/asset.py", line 97, in prepare_depreciation_data
18:09:16 web.1            |     self.make_depreciation_schedule(date_of_disposal, value_after_depreciation)
18:09:16 web.1            |   File "apps/erpnext/erpnext/assets/doctype/asset/asset.py", line 339, in make_depreciation_schedule
18:09:16 web.1            |     self._make_depreciation_schedule(
18:09:16 web.1            |   File "apps/erpnext/erpnext/assets/doctype/asset/asset.py", line 430, in _make_depreciation_schedule
18:09:16 web.1            |     and get_updated_rate_of_depreciation_for_wdv_and_dd(
18:09:16 web.1            |   File "apps/erpnext/erpnext/__init__.py", line 151, in caller
18:09:16 web.1            |     return frappe.get_attr(overrides[function_path][-1])(*args, **kwargs)
18:09:16 web.1            | TypeError: get_updated_rate_of_depreciation_for_wdv_and_dd() takes 3 positional arguments but 4 were given
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SherinKR** [comment]
@vorasmit It's got fixed in #1392 Please make the relase as it's causing a major road block
Thankyou

**Comment by vorasmit** [comment]
Thanks for reporting the issue. We just released it to version-14 and 15
